AgContext.yaml

```diff
@@ -0,0 +1,321 @@
+type: object
+required:
+  - id
+  - crop_type
+  - crop_stage
+  - soil_colour
+  - surface_cover
+  - surface_coverage
+  - weather_description
+  - location_lat
+  - location_long
+  - location_datum
+  - camera_model
+  - lens_model
+  - camera_height
+  - camera_angle
+  - camera_fov
+  - photography_description
+  - cropped_to_plant
+
+properties:
+
+  id:
+    type:
+      number
+    description: |-
+        A number identifying the AgContext.
+
+  crop_type:
+    type:
+      string
+    description: |-
+          General crop type.
+          A string describing the general cropping scenario (e.g. "root vegetable").
+
+  grains_descriptive_text:
```

Comment:
I currently have the three different crop stage descriptions in here. Is it possible to link them to each other, or is that over engineering for the schema?

Reply:
As per bbch_descriptive_text, I don't think these should belong here.

Ideally we'd get to the stage of mocking up a search form soon. Maybe you and I should discuss how we'd get there.